aicsimageio/readers/tiff_reader.py

```diff
@@ -130,7 +130,10 @@ def _read_delayed(self) -> da.core.Array:
             # Combine length of scenes and operating shape
             # Replace YX dims with empty dimensions
             operating_shape = (len(scenes), *operating_shape)
-            operating_shape = operating_shape[:-2] + (1, 1)
+            if scenes[0].keyframe.samplesperpixel != 1:  # if it's a multichannel (RGB)
```

Comment:
Also, rather than scene 0, based on the code above this, it might be slightly better to use `self.specific_s_index`

Reply:
I don't believe that "S" will ever be returned from axes? Could be totally wrong but "S" is `scenes`. Each `scene` isn't going to contain itself as an ax -- this is why my original code says the operating shape is `(len(scenes), *operating_shape)` i.e. `("S", *rest_of_dims)`

Reply:
This is more so just `YXC`? where C is the RGB no?

Reply:
As I understand it I would have called it C but for some reason, they map that to S. This could be a mess down the road, how does this work when we have a multiscene RGB tiff. It might work but I'm not entirely sure.

Reply:
S and C are different things.  This S is specific to the TIFF format.  it's samples per pixel.  it's there for encoding rgb data.

Reply:
OH. This is like an actual dimension "S" not "Scenes"???

Reply:
The point that S in tifffile represents Samples is true but we have to put it in a dimension so it's either S or C and to me C makes more sense. I'm still concerned we don't have a test file with multiple channels that's RGB.

Reply:
it's very possible to have such a thing.  "Channels" are just an ordering of the tiff planes, but "Samples" refer to the data inside an xy plane.

Reply:
So is your intention that when you load a RGB tiff that has multiple channels N_C, you get N_C*3 channels in your numpy array?  Is that what the non-dask code path returns?